p2p/protocol.py

```diff
@@ -28,10 +32,12 @@
 if TYPE_CHECKING:
     from p2p.peer import BasePeer  # noqa: F401
 
+TypedDictPayloadType = TypedDict("TypedDictPayloadType", {})
```

Comment:
nitpick: `TypedPayloadDict` would be more inline with the rest of the names in the code base. In general, we don't use a `Type` suffix for type names except we try to express that something is modelling the *kind* / *category* / *type* of something.

e.g.

```python
class CarType(enum):
    SportsCar = auto()
    Van = auto()
```

But apart from such cases we would have classes such as `Animal`, `Building`, `Food` and not `AnimalType`, `BuildingType` or `FoodType`.